imap_processing/cdf/config/imap_idex_l1_variable_attrs.yaml

```diff
@@ -0,0 +1,420 @@
+int_fillval: &int_fillval -9223372036854775808
+min_epoch: &min_epoch -315575942816000000
+max_epoch: &max_epoch 3155630469184000000
+
+data_min: &data_min 0
+data_max: &data_max 4096
```

Comment:
I don't see these being used anywhere else in the file. Could they be removed?

Reply:
Just double checked `idex_cdf_attributes.py` from dev branch, and these are supposed to be used for l1_data_base. My bad!